v5/api/cpp/motors.rst

```diff
@@ -2347,52 +1924,11 @@ Indicates the internal gearing used by the motor.
  pros::E_MOTOR_GEARSET_INVALID      Error return code
 ================================== ===========================================================
 
-Typedefs
-========
-
-motor_pid_full_s_t
-------------------
-
-Holds the information about a Motor's position or velocity PID controls.
-
-These values are in 4.4 format, meaning that a value of 0x20 represents 2.0,
-0x21 represents 2.0625, 0x22 represents 2.125, etc.
-
-::
-
-  typedef struct motor_pid_full_s {
-    uint8_t kf;        // The feedforward constant
-    uint8_t kp;        // The proportional constant
-    uint8_t ki;        // The integral constants
-    uint8_t kd;        // The derivative constant
-    uint8_t filter;    // A constant used for filtering the profile acceleration
-    uint16_t limit;    // The integral limit
-    uint8_t threshold; // The threshold for determining if a position movement has
-                       // reached its goal. This has no effect for velocity PID
-                       // calculations.
-    uint8_t loopspeed; // The rate at which the PID computation is run in ms
-  } motor_pid_full_s_t;
-
-----
-
-motor_pid_s_t
--------------
-
-Holds just the constants for a Motor's position or velocity PID controls.
-
-These values are in 4.4 format, meaning that a value of 0x20 represents 2.0,
-0x21 represents 2.0625, 0x22 represents 2.125, etc.
-
-::
-
-  typedef struct motor_pid_s {
-    uint8_t kf;        // The feedforward constant
-    uint8_t kp;        // The proportional constant
-    uint8_t ki;        // The integral constants
-    uint8_t kd;        // The derivative constant
-  } motor_pid_s_t;
-
-
 .. _motor_gearset_e_t: ../c/motors.html#motor-gearset-e-t
 .. _motor_encoder_units_e_t: ../c/motors.html#motor-encoder-units-e-t
 .. _motor_brake_mode_e_t: ../c/motors.html#motor-brake-mode-e-t
+
+Typedefs
+======
+
+None.

```

Comment:
The typedefs (gearset, encoder, and brake) enums are still here... you should move these under the typedefs section.

Reply:
(Or remove the typedefs, forgot enums are its own section)